### PR TITLE
bpo-32255: Fix inconsistent behavior when csv.writer writes None

### PR DESCRIPTION
**Pull request by Licht-T**
See this.
https://github.com/pandas-dev/pandas/issues/18676
https://bugs.python.org/issue32255

I am waiting for the CLA response.

https://bugs.python.org/issue32255

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately our records indicate you have not signed the CLA. For legal reasons we need you to sign this before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.


Thanks again to your contribution and we look forward to looking at it!

File: Lib/test/test_csv.py [hunk 1]
```diff
@@ -211,6 +211,25 @@ def write(self, buf):
             fileobj.seek(0)
             self.assertEqual(fileobj.read(), "a,b\r\nc,d\r\n")
 
+    def test_writerows_with_none(self):
+        with TemporaryFile("w+", newline='') as fileobj:
+            writer = csv.writer(fileobj)
+            writer.writerows([['a',None],[None,'d']])
```
Comment (serhiy-storchaka) [inline Lib/test/test_csv.py]: Please add spaces after commas to conform PEP 8.

**Review by serhiy-storchaka — COMMENTED** [review]
LGTM besides a small style nitpick.

Please add a `Misc/NEWS.d/` entry (manually or using the `blurb` tool, see the [devguide](https://devguide.python.org/committing/#what-s-new-and-news-entries)) and add your name into `Misc/ACKS`.

**Comment by Licht-T** [comment]
Thanks @serhiy-storchaka, fixed!

File: Misc/NEWS.d/next/Library/2017-12-12-07-29-06.bpo-32255.2bfNmM.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+Fix inconsistent ``csv.writer`` behavior when it handles ``None``
```
Comment (serhiy-storchaka) [inline Misc/NEWS.d/next/Library/2017-12-12-07-29-06.bpo-32255.2bfNmM.rst]: Add "Patch by Licht Takeuchi."

Comment (serhiy-storchaka) [reply Misc/NEWS.d/next/Library/2017-12-12-07-29-06.bpo-32255.2bfNmM.rst]: And don't forgot periods at the end of sentences.

**Comment by Licht-T** [comment]
@serhiy-storchaka Fixed!

**Review by serhiy-storchaka — APPROVED** [review]

**Pull request merged**